de/cheatsheet-deep-learning.md

```diff
@@ -162,160 +162,160 @@
 
 **28. LSTM ― A long short-term memory (LSTM) network is a type of RNN model that avoids the vanishing gradient problem by adding 'forget' gates.**
 
-&#10230;
+&#10230; LSTM - Ein langes Kurzzeitgedächtnis (long short-term memory, LSTM) ist ein Typ eines RNN welcher durch Hinzufügen von Vergesstoren das Problem verschwindender Gradienten vermeidet. 
 
 <br>
 
 **29. Reinforcement Learning and Control**
 
-&#10230;
+&#10230; Bestärkendes Lernen und bestärkende Regelung
 
 <br>
 
 **30. The goal of reinforcement learning is for an agent to learn how to evolve in an environment.**
 
-&#10230;
+&#10230; Das Ziel des bestärkenden Lernens ist für einen Agenten zu lernen sich in einer Umgebung zu entwickeln.
```

Comment:
Das Ziel des bestärkenden Lernens ist ~~für einen Agenten zu lernen~~ **einen Agenten zu traineren welcher selbstständig lernt** sich in einer Umgebung zu entwickeln.